README.md

```diff
@@ -0,0 +1,41 @@
+# mender-push-artifact
+
+A GitHub action to push your artifacts to a hosted Mender instance.
+
+## Features / Limitations
+
+The action supports uploading all artifact versions (v1, v2, v3) to an arbitrary hosted Mender instance.
+
+A "personal access token" are used as authentication mechanism. Please read [the documentation](https://docs.mender.io/server-integration/using-the-apis#personal-access-tokens) on how to create one for your Github Actions pipeline setup.
+
+## Usage
+
+This action require the following inputs:
+
+- `mender_pat`: personal access token on the hosted Mender instance to be used
```

Comment:
We should call it `jwt` as we do in all over our documentation

Reply:
I thought quite a lot about that and still not sure what is better approach here. From one side it's in fact `jwt` token, but from another side it might be misleading and confusing, because personal access tokens is a separate feature and `pat` here explicitly highlights that.

The confusion might come from using a JWT token ([/api/management/v1/useradm/auth/login](https://docs.mender.io/api/#management-api-user-administration-and-authentication-login) API call), as described in our [Using the APIs](https://docs.mender.io/server-integration/using-the-apis#install-curl-and-jq-and-set-up-the-shell-variables) docs section, which expires fast, instead of explicitly issued for CI/CD purposes long running personal access token.